base_conditional_image/views/image_view.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+  <data>
```

Comment:
Entire file is bad intended (should be 4 spaces)

Reply:
@acsonefho AFAIR, there is no common agreement on the tab space into OCA repos.

Reply:
Ok ok no problem, I thought there was